src/assets/data/localisations.ts

```diff
@@ -1475,5 +1475,35 @@ export const Localisations = {
     nl: 'Tekst invoeren',
     pl: 'Wprowadź tekst',
     hb: 'הזן טקסט'
+  },
+  PROGRESS_REPORT_TITLE: {
+    da: '',
```

Comment:
Maybe something optional would be to have some rough translations here from Google translate?

Reply:
Yes, I will do this.